src/code/actions.js

```diff
@@ -275,9 +363,33 @@ export function initiateDelayedFertilization(delay, gamete1, gamete2) {
   };
 }
 
-export function keepOffspring() {
+export function _keepOffspring(index, success) {
```

Comment:
You're right, thanks @knowuh. 

In theory they could also be public, but we don't yet have a good naming distinction between actions that just do a tiny piece of a larger action, and the larger actions that you would actually want to call.

Probably the two "_littleActions" I just un-exported now in 00a8935 could not be action-creator functions at all, and instead the bigger actions could just dispatch the action object itself. Not sure -- we'll probably continue to change conventions as we go on.